packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5407,6 +5441,25 @@ class InheritedElement extends ProxyElement {
     setDependencies(dependent, null);
   }
 
+  /// Called by dependent widgets when they stop listening to this [InheritedElement].
+  ///
+  /// This either happens when the dependent is unmounted, or, if
+  /// [clearDependencyOnRebuild] is true, before the dependent rebuilds.
+  ///
+  /// See also:
+  ///
+  ///  * [getDependencies], which returns the current value for a dependent
+  ///    element.
+  ///  * [setDependencies], which sets the value for a dependent element.
+  ///  * [notifyDependent], which can be overridden to use a dependent's
+  ///    dependencies value to decide if the dependent needs to be rebuilt.
+  ///  * [InheritedModel], which is an example of a class that uses this method
+  ///    to manage dependency values.
+  @protected
```

Comment:
> Looking at it, the other methods do not have such annotation. So not sure if it really makes sense

What other methods do you mean?

I would argue that this one needs it because if you forget it you actually have no way of fulfilling the contract of the method since `_dependents` is private.

Reply:
`getDependencies`/`setDependencies`

They both refer to `_dependents` too, but don't have `mustCallSuper`